app/views/confirmations/_sent/_confirm_change.html.erb

```diff
@@ -0,0 +1,31 @@
+<% content_for :title, t("confirmation_sent.heading.confirm_change") %>
+
+<%= render "govuk_publishing_components/components/heading", {
+  text: yield(:title),
+  heading_level: 1,
+  margin_bottom: 4,
+  font_size: "xl",
+} %>
+
+<%= sanitize(t("confirmation_sent.instructions.confirm_change.details.#{returning_service}", email: email)) %>
```

Comment:
`returning_service` will always be `:dashboard` here, because you can't change your email from something else.